scipy/optimize/_linprog_highs.py

```diff
@@ -200,14 +208,17 @@ def _linprog_highs(lp, solver, time_limit=None, presolve=True,
     _check_unknown_options(unknown_options)
 
     # Map options to HiGHS enum values
-    simplex_dual_edge_weight_strategy = _convert_to_highs_enum(
+    if simplex_dual_edge_weight_strategy is None:
+        simplex_dual_edge_weight_strategy = 'steepest-devex'
```

Comment:
Here's where `simplex_dual_edge_weight_strategy=None` is converted to `'steepest-devex'`

Reply:
> @mdhaber I am satisfied to start asking for reviews again. The API has been greatly simplified and we've got a few more checks in place to make sure things behave as expected. I'm not sure on the timeline for `start_crossover_tolerance` because it seems to be tied to another larger refactor in the works (see [ERGO-Code/HiGHS#403](https://github.com/ERGO-Code/HiGHS/pull/403)). It will either turn out to be a small wrapper addition now or an additional option introduced later so I'll let you decide if it's worth waiting for that option or if it will be okay to add the option later on.

I've now done what I should have done - created a branch from master for these minor options changes, and re-implemented the introduction of start_crossover_tolerance. I've also introduced HIGHS_SIMPLEX_DUAL_EDGE_WEIGHT_STRATEGY_CHOOSE and HIGHS_SIMPLEX_PRIMAL_EDGE_WEIGHT_STRATEGY_CHOOSE, and got rid of SIMPLEX_DUAL_EDGE_WEIGHT_STRATEGY_STEEPEST_EDGE_TO_DEVEX_SWITCH as this is accomplished by HIGHS_SIMPLEX_DUAL_EDGE_WEIGHT_STRATEGY_CHOOSE. 

These changes are now pushed to master

Reply:
Thanks @jajhall!  I have them integrated [here](https://github.com/mdhaber/scipy/pull/42)